anda/apps/micro/micro-nightly.spec

```
@@ -0,0 +1,74 @@
+# https://github.com/zyedidia/micro
+%global goipath        github.com/zyedidia/micro
+%global commit         2259fd10affd19be60400a1ac2c86b920f2c64c2
+
+%global shortcommit    %(c=%{commit}; echo ${c:0:7})
+%global commit_date    20240731
+%global compiledate    %(date "+%%B %%d, %%Y")
+
+%gometa -f
+
+%global goname         micro
+
+%global common_description %{expand: 
+Micro is a terminal-based text editor that aims to be easy to use and intuitive, while also taking advantage of the full capabilities of modern terminals.}
+
+%global golicenses     LICENSE LICENSE-THIRD-PARTY
+%global godocs         README.md
+
+Name:           %{goname}-%{tag}
+Version:        2.0.14~dev^%{commit_date}g%{shortcommit}
+Release:        %autorelease
+Summary:        A modern and intuitive terminal-based text editor
+License:        MIT AND Apache-2.0 AND MPL-2.0
+URL:            https://micro-editor.github.io
+Source0:        %{gosource}
```

Oh yeah, by the way, we are currently updating the Terra submission policies and guidelines, and you are encouraged to add yourself to `Packager:`
```suggestion
Source0:        %{gosource}
Packager:       davidusrex <dlunn@outlook.com>
```